AirCasting/CreateSessionViews/CreateSessionDetailsView.swift

```diff
@@ -143,7 +143,7 @@ private extension CreateSessionDetailsView {
 
     func getAndSaveStartingLocation() {
         let fakeLocation = CLLocationCoordinate2D(latitude: 200.0, longitude: 200.0)
-        if isIndoor {
+        if isIndoor && sessionContext.sessionType == .fixed {
```

Comment:
Mobile sessions can't be indoor so it's not necessary, on the other hand, we can leave it as it is to make sure that we're not sending fake locations for mobile sessions flagged as indoor by mistake 🤔

Reply:
turns out this flag starts as `true` in this view and the switch is not visible for mobile sessions so the execution was indeed getting here and sending false location (200/200) to server :) I consider this a hotfix because all this should really be handled elsewhere